engine/hexen2/Makefile

```diff
@@ -992,9 +992,11 @@ USE_CDAUDIO=no
 endif
 
 ifndef DEBUG
-CFLAGS  += -O3
+#CFLAGS  += -O3
 # -fomit-frame-pointer / -ffast-math causes trouble with gcc2.95
-CFLAGS  += -fno-omit-frame-pointer
+#CFLAGS  += -fno-omit-frame-pointer
+# these break the game with GCC6
+CFLAGS  += -fbbb=- -fno-tree-forwprop
```

Comment:
These aren't supported by old gcc2.95: We need some flag or some makefile variable to support it?

Reply:
I can add an variable e.g. GCC = 6, but I found that GCC6 usually produces better code than older versions, so I thought it's enough to keep the old flags commented out in case someone really wants to use GCC2.

Reply:
Some `AMIGA_GCC6`, or `BEBBO_TOOLCHAIN`, or something may be good. (I don't have bebbo's toolchain available on my setup yet, that why I am whining :))

Reply:
I'll probably go with BEBBO_TOOLCHAIN then, since he improved a lot of things besides GCC (e.g. libnix).

Reply:
OK